src/docs.md

```diff
@@ -378,6 +378,15 @@ To invalidate caches and reload:
 cadenzaClient.reload({ invalidateCaches: true });
 ```
 
+### Close itself
+
+<small>API: [CadenzaClient#closeMe](./classes/CadenzaClient.html#reload)</small>
```

Comment:
The method mentioned here is wrong, it should reference closeMe but references reload